README.md

```diff
@@ -117,6 +117,13 @@ chatkit_integration_locator=<INSTANCE_LOCATOR>
 chatkit_integration_key=<SECRET_KEY>
 ```
 
+## Linting
+
+We currently use [ktlint](https://github.com/pinterest/ktlint). You can use `ktlintCheck` to check if there are any errors. Most errors can be automatically resolved by using `ktlintFormat` - if it can't auto resolve the issue, it will let you know what to do.
+
+You may want to configure your Android Studio to more closely match what ktlint expects:
+* wildcards - to disable Android Studio using wildcard imports head to settings - editor - code style - kotlin - imports (it's a tab) - and select use single name import for top level imports and java statics and enum members.
```

Comment:
We should configure it for the project and check in the generated `.idea/codeStyles/Project.xml`. Doesn't ktlint provide the IDE config matching theirs?

Reply:
Good point - I followed the instructions at https://github.com/pinterest/ktlint#-with-intellij-idea (following option 1) which was `ktlint --android applyToIDEAProject` however it didn't produce anything that i could commit. I need to look at what our gitignore file excludes.

Reply:
It said it updated:

```
The following files are going to be updated:
	./.idea/codeStyles/codeStyleConfig.xml
	./.idea/codeStyles/Project.xml
	./.idea/inspectionProfiles/profiles_settings.xml
	./.idea/inspectionProfiles/ktlint.xml
	./.idea/workspace.xml
```

Reply:
Looks like our .gitignore contains `/.idea/*` so i'm not sure if that's the thing that's hitting it :/

Reply:
Have updated our .gitignore to include the code styling files and excluded things listed from the [github android gitignore suggestions](https://github.com/github/gitignore/blob/master/Android.gitignore)